src/PresentationExchangeAgent.ts

```diff
@@ -0,0 +1,40 @@
+import { EvaluationResults, PEJS, SelectResults, Validated } from '@sphereon/pe-js';
```

Comment:
Nice to have the PE Agent :) It is okay for me to have a bit of AuthReq knowledge in here, as we are otherwise basically delegating everything to pejs anyway